src/docs/sdk/performance/span-operations.mdx

```diff
@@ -16,7 +16,7 @@ It's important to keep categories consistent between SDKs and integrations as th
 
 The following tables contain examples of operations used by the SDKs and Sentry product. The usage column in the table contains examples of using that operation category, but are not hard recommendations for operation usage. As long as categories stay consistent, SDK developers are free to choose actions and identifiers that best match the use case they are instrumenting.
 
-If a span operation is not provided, the value of `default` is set by Relay.
+If a span operation is not provided, the value of `default` is set by Relay. The value `default` can also be set by SDKs, for example to avoid breaking code when no longer requiring `op` to be set on `TransactionContext`.
```

Comment:
```suggestion
If a span operation is not provided, the value of `default` is set by Relay. The value `default` can also be set by SDKs, for example, to avoid breaking changes when no longer requiring `op` to be set on `TransactionContext`.
```